config/services.yml

```diff
@@ -1016,9 +996,43 @@ https://cmr.uat.earthdata.nasa.gov:
         conditional:
           format: ['application/x-zarr']
 
-  - name: sds/HOSS-maskfill
-    # Provides polygon spatial subsetting for collections hosted in OPeNDAP
-    # Also provides variable and temporal subsetting through HOSS.
+  - name: sds/HOSS-geographic
+    # Provides variable, temporal, bounding box spatial and polygon spatial
+    # subsetting for geographically gridded collections hosted in OPeNDAP.
+    data_operation_version: '0.17.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/HOSS
+    umm_s:
+      - S1240682712-EEDTEST
+      - S1241070355-GES_DISC # GESDISC_HOSS
+      - S1246887053-ORNL_CLOUD
+    collections: []
+    capabilities:
+      subsetting:
+        bbox: true
+        dimension: true
+        shape: true
+        variable: true
+      output_formats:
+        - application/netcdf # Incorrect mime-type, remove when no longer needed
+        - application/x-netcdf4
+    steps:
+      - image: !Env ${QUERY_CMR_IMAGE}
+      - image: !Env ${VAR_SUBSETTER_IMAGE}
+        operations: ['variableSubset', 'spatialSubset', 'dimensionSubset', 'shapefileSubset']
+      - image: !Env ${SDS_MASKFILL_IMAGE}
+        operations: ['shapefileSubset']
+        conditional:
+          exists: ['shapefileSubset']
```

Comment:
This and L1063 are the main difference between these two service chains:

* If a collection is geographically gridded, MaskFill should only be called for a shape file spatial subset, because a bounding box should line up with the spatial dimensions of the grid. This means HOSS alone, getting a rectangular patch from OPeNDAP is sufficient.
* If a collection is projection-gridded (non-geographic), MaskFill will be needed for any spatial subset, including a bounding box, because latitude and longitude will not line up with the grid dimensions, so some of the rectangular HOSS output will need to be filled for a bounding box spatial subset.